github_connector_odoo/models/odoo_category.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Oscar Alcala (https://twitter.com/oscarolar)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+class OdooCategory(models.Model):
+    _name = 'odoo.category'
+
+    name = fields.Char()
```

Comment:
Can you add a simple string='Name' ? 
Makes the field more userfriendly

Reply:
Odoo's ORM will default the string to `Name` wherever the field is shown.